lang/python/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Use the official Python base image
+FROM --platform=linux/amd64 python:3.11-rc-slim-buster as pythonbaseimage
+
+RUN pip install poetry
+
+
+ENV PATH="/root/.poetry/bin:${PATH}"
+
+COPY ./xtdb_juxt ./xtdb_juxt
+COPY ./pyproject.toml .
+COPY ./README.md .
+
+RUN poetry install
```

Comment:
Small note, if you make this:
```Dockerfile
COPY ./pyproject.toml .
RUN poetry install

COPY ./xtdb_juxt ./xtdb_juxt
```

Then docker will only re-run `poetry install` (which can take a while) when the pyproject.toml changes.
Not super important because I think github actions doesn't cache image layers, but nice for local development :)